src/class-parsely.php

```diff
@@ -438,7 +438,13 @@ public function initialize_settings() {
 			)
 		);
 
-		$h = __( 'The plugin uses http canonical URLs by default. If this needs to be forced to use https, set this option to true. Note: the default is fine for almost all publishers, it\'s unlikely you\'ll have to change this unless directed to do so by a Parse.ly support rep.', 'wp-parsely' );
+		$parsely_options = $this->get_options();
+		if ( $parsely_options['force_https_canonicals'] === $this->is_home_url_with_https() ) {
+			$h = __( 'This option is being consistent with the protocol of your site address. It\'s unlikely you\'ll have to change this unless directed to do so by a Parse.ly support rep.', 'wp-parsely' );
```

Comment:
What if we changed this to:

> We've detected that your site uses https so we've set this option for you automatically!
> If you need to change this option, please speak to a Parse.ly customer support representative so we can properly index your site along with the change.

What do you think?

Reply:
> We've detected that your site uses https so we've set this option for you automatically!

This statement is not really correct for all cases. 
It's correct only if: 

```php
$parsely_options['force_https_canonicals'] === $this->is_home_url_with_https() === true
``` 

 but it's not correct for the other case: 
```php
$parsely_options['force_https_canonicals'] === $this->is_home_url_with_https() === false
``` 
Therefore, I think it would be better to stick with the "consistent" word.

> If you need to change this option, please speak to a Parse.ly customer support representative so we can properly index your site along with the change.

Actually, I prefer my current wording (also the previous wording before this PR) as it indicates clearly everything still works properly and customers do not need to do anything.

Reply:
Otherwise, in case, we need to state which option it's being right now (HTTP or HTTPS) in the help text, I can add more code to handle that. However, the downside is that the code may be verbose and not necessary in terms of telling what's next for users. 

What do you think? I do not have a strong opinion regarding this, just want to balance between "detailed info" and "simple code".

Reply:
Thanks for your thoughts @htdat!  I don't have any strong opinions, but curious to see what @GaryJones thinks about the wording for these messages.